lamina-c/Makefile

```
@@ -0,0 +1,110 @@
+
+all: $(STABLE_BINS)
+
+# lib runs cargo and touches a stamp file on success.
+# Examples depend on the stamp so they relink only when cargo produces new output.
+lib: $(LIB_STAMP)
+
+$(LIB_STAMP):
+	$(CARGO) build $(CARGO_PROFILE_FLAG)
+	touch $(LIB_STAMP)
+
+$(LIB_FILE): $(LIB_STAMP)
```

checkmake (0.3.2)

[warning] Target "$(LIB_FILE)" should be declared PHONY.

(phonydeclared)